bin/localeapp

```diff
@@ -68,6 +68,7 @@ command :install do |c|
     installer.config_type = :standalone if options[:standalone]
     installer.config_type = :heroku if options[:heroku]
     installer.config_type = :github if options[:github]
+    installer.write_dotenv = true if File.exist?('.env')
```

Comment:
Command line switch (`-d` / `--dotenv`) has been added.

As the feature is now opt-in (nothing happens unless you use the switch), detection if the `.env` file exists has been dropped altogether.